src/components/docs-layout.jsx

```diff
@@ -71,6 +72,16 @@ export default function DocumentPage({ children, metadata }) {
 				</nav>
 				<nav className="sticky top-[84px] order-last hidden h-[calc(100vh-84px)] w-[240px] overflow-y-auto p-6 lg:block">
 					<OnThisPageNav>{children}</OnThisPageNav>
+
+					<div className="my-6 border-t border-gray-700" />
+
+					<Link
+						className="font-normal text-gray-400 no-underline hover:text-blue-500"
+						href={`https://github.com/wpengine/faustjs.org/edit/main/src/pages${asPath}index.mdx`}
+						noDefaultStyles
+					>
+						Edit this doc on GitHub
```

Comment:
@colinmurphy What do you think about this phrase:

"Improve & Edit this doc on GitHub"

Reply:
@Fran-A-Dev I really like that suggestion 😄 

Let me get a update for the PR.

Reply:
@Fran-A-Dev 

Pushed a change for this 😄